Content.Client/Power/PowerMonitoringWindow.xaml.Widgets.cs

```diff
@@ -104,6 +105,28 @@ public void UpdateWindowEntryButton(NetEntity netEntity, PowerMonitoringButton b
         // Update power value
         // Don't use SI prefixes, just give the number in W, so that it is readily apparent which consumer is using a lot of power.
         button.PowerValue.Text = Loc.GetString("power-monitoring-window-button-value", ("value", Math.Round(entry.PowerValue).ToString("N0")));
+
+        // Update battery level if applicable
+        if (entry.BatteryLevel != null)
+        {
+            var batteryPercentage = (int)(entry.BatteryLevel.Value * 100);
+
+            button.BatteryLevel.Value = batteryPercentage;
+            button.BatteryLevel.Visible = true;
+
+            button.BatteryPercentage.Text = $"{batteryPercentage}%";
+            button.BatteryPercentage.Visible = true;
+
+            // Set progress bar color based on percentage
+            var color = Color.FromHsv(new Vector4((batteryPercentage / 100f) * 0.33f, 1, 1, 1));
```

Comment:
Why not use `entry.BatteryLevel.Value` directly? You're dividing and then multiplying by 100 here.